### DISCUSSION TITLE
Unfortunately, we weren't able to approve your educational discount request

### DISCUSSION THREAD

**Discussion by phuntrb** [community/community#18830] [GitHub Education]
I tried register Studen Github, but it was rejected many times :(
Please help me!!!

<img width="885" alt="image" src="https://user-images.githubusercontent.com/107702929/174420504-47cf06b8-663b-40ec-a1ff-89d50bad070b.png">

BTW, the image I uploaded doesn't modified, so the error message from Github team very confuse
So sad!!!!

**Comment by aenq** [comment]
Reapply and make sure it meets the requirements
https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/why-wasnt-my-application-for-a-student-developer-pack-approved
